content/blog/2021-05-24-radiobeitrag-deutschlandfunknova/index.de.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Der OpenBikeSensor auf Deutschlandfunk
+linkTitle: Der OpenBikeSensor auf Deutschlandfunk
```

Comment:
This linkTitle is a bit long for  the sidebar navigation. At some point we should rethink that navigation, but until then please shorten this, e.g. "Deutschlandfunk-Beitrag"

Reply:
Fixed